torchao/dtypes/uintx/plain_layout.py

```diff
@@ -253,16 +253,25 @@ def _linear_fp_act_int8_weight_impl(input_tensor, weight_tensor, bias):
     # return torch.ops.aten._weight_int8pack_mm(input_tensor.contiguous(), w_vals_int8_t, weight_tensor.tensor_impl.scale)
 
     # per channel int8 weight only quantizated mm
-    w_vals_int8_t = weight_tensor.tensor_impl.int_data.t()
+    w_vals_int8 = weight_tensor.tensor_impl.int_data
```

Comment:
this is a code path for int8 cuda as well I think, changing it has a risk of perf regressions

also this is the older stack, I'd suggest to migrate first, WIP here for int8 + plain layout: https://github.com/pytorch/ao/pull/3038